packages/task/src/common/task-protocol.ts

```diff
@@ -177,8 +177,6 @@ export interface TaskIdentifier {
 export interface TaskInfo {
     /** internal unique task id */
     readonly taskId: number,
-    /** terminal id. Defined if task is run as a terminal process */
-    readonly terminalId?: number,
```

Comment:
Why are we removing the `terminalId` from the interface.
It is an unnecessary breaking change.